server/services/message-service.js

```diff
@@ -1,38 +1,32 @@
-const amqp = require('amqplib/callback_api')
-const scheduleService = require('./schedule-service')
+const container = require('rhea')

```

Comment:
as @devotee101 mentioned, promises are preferred to callbacks so I think it would be worth migrating across to use rhea promises. Both libraries are maintained by AMQP, so they should have the same features.

Reply:
Looks like promises would leave us with cleaner looking code. I'll do that to this service, first.